app/index.html

```diff
@@ -4,10 +4,32 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/reset.css">
```

Comment:
- [ ] I believe that you can delete this link tag, what do you think? because you already specify the stylesheet link for `CSS` in another link ` <link rel="stylesheet" href="styles/main.css">`

Reply:
Deleted the link tag.